scripting/shavit-misc.sp

```diff
@@ -2085,6 +2092,92 @@ public Action Shotgun_Shot(const char[] te_name, const int[] Players, int numCli
 	return Plugin_Stop;
 }
 
+public Action EffectDispatch(const char[] te_name, const Players[], int numClients, float delay)
+{
+	int iEffectIndex = TE_ReadNum("m_iEffectName");
+	int nHitBox = TE_ReadNum("m_nHitBox");
+	char sEffectName[64];
+
+	GetEffectName(iEffectIndex, sEffectName, 64);
+
+	if(gB_NoBlood)
```

Comment:
Could you move this `if()` statement to the beginning of `EffectDispatch` and invert it? So it looks like this:

```sp
if(!gB_NoBlood)
{
	return Plugin_Continue;
}
```